tests/contract_testcases/solana/bad_seeds_on_external_call.sol

```diff
@@ -26,10 +26,10 @@ contract c {
 
 // ---- Expect: diagnostics ----
 // error: 13:4-7: type bytes1 found where array of slices expected
-// error: 15:4-5: expected 'bytes[]', found integer
+// error: 15:4-5: expected 'bytes slice[]', found integer
```

Comment:
What is `bytes slice[]` supposed to be in Solidity?
I think the error message should point to something one can write in Solidity. You are providing a hint that does not work.

Reply:
I've removed `slice` from diagnostics and now it looks like solidity types.